content/events/release-apache-skywalking-apm-9.2.0/index.md

```diff
@@ -0,0 +1,143 @@
+* Fix race condition in Banyandb storage
+* Support `SUM_PER_MIN` downsampling in `MAL`.
+* Support `sumHistogramPercentile` in `MAL`.
+* Add `VIRTUAL_CACHE` to Layer, to fix conjectured Redis server, which icon can't show on the topology.
+* [Breaking Change] Elasticsearch storage merge all metrics/meter and records(without super datasets) indices into one
+  physical index template `metrics-all` and `records-all` on the default setting.
+  Provide system environment variable(`SW_STORAGE_ES_LOGIC_SHARDING`) to shard metrics/meter indices into
+  multi-physical indices as the previous versions(one index template per metric/meter aggregation function).
+  In the current one index mode, users still could choose to adjust ElasticSearch's shard
+  number(`SW_STORAGE_ES_INDEX_SHARDS_NUMBER`) to scale out.
+  More details please refer to [New ElasticSearch storage option explanation in 9.2.0](../FAQ/New-ElasticSearch-storage-option-explanation-in-9.2.0.md)
+  and [backend-storage.md](../setup/backend/backend-storage.md)
+* [Breaking Change] Index/table `ebpf_profiling_schedule` added a new column `ebpf_profiling_schedule_id`,
+  the H2/Mysql/Tidb/Postgres storage users are required to re-created it when bump up from previous releases.
+* Fix Zipkin trace query the max size of spans.
+* Add `tls` and `https` component IDs for Network Profiling.
+* Support Elasticsearch column alias for the compatibility between storage logicSharding model and no-logicSharding model.
+* Support MySQL monitoring.
+* Support PostgreSQL monitoring.
+* Fix query services by serviceId error when Elasticsearch storage `SW_STORAGE_ES_QUERY_MAX_SIZE` > 10000.
+* Support sending alarm messages to Discord.
+* Fix query history process data failure.
+* Optimize TTL mechanism for Elasticsearch storage, skip executed indices in one TTL rotation.
+* Add Kubernetes support module to share codes between modules and reduce calls to Kubernetes API server.
+* Bump up Kubernetes Java client to fix cve.
+* Adapt OpenTelemetry native metrics protocol.
+* [Breaking Change] rename configuration folder from `otel-oc-rules` to `otel-rules`.
+* [Breaking Change] rename configuration field from `enabledOcRules` to `enabledOtelRules` and
+  environment variable name from `SW_OTEL_RECEIVER_ENABLED_OC_RULES` to `SW_OTEL_RECEIVER_ENABLED_OTEL_RULES`.
+* [Breaking Change] Fix JDBC TTL to delete additional tables data. 
+  SQL Database requires removing `segment`,`segment_tag`, `logs`, `logs_tag`, `alarms`, `alarms_tag`, `zipkin_span`, `zipkin_query` before OAP starts.
+* SQL Database: add `@SQLDatabase.ExtraColumn4AdditionalEntity` to support add an extra column from parent to an additional table.
+* Add component ID(131) for Java Micronaut plugin
+* Add component ID(132) for Nats java client plugin
+
+
+#### UI
+
+* Fix query conditions for the browser logs.
+* Implement a url parameter to activate tab index.
```

Comment:
```suggestion
* Implement a URL parameter to activate tab index.
```

Reply:
This is copied from the original change log :)